app/pages/home.cjsx

```diff
@@ -79,7 +79,7 @@ module.exports = React.createClass
     document.documentElement.classList.remove 'on-home-page'
 
   lastFourProjects: ->
-    @props.user.get("project_preferences", page_size: 4, sort: '+updated_at')
+    @props.user.get("project_preferences", page_size: 4, sort: 'updated_at')
```

Comment:
I'm not sure that's right. To me it looks like this (and `-updated_at`) returns four oldest. `+updated_at` returns the four most recent.